cmd/kubeadm/app/phases/kubelet/flags.go

```diff
@@ -72,8 +73,12 @@ func buildKubeletArgMap(nodeRegOpts *kubeadmapi.NodeRegistrationOptions, registe
 		kubeletFlags["register-with-taints"] = strings.Join(taintStrs, ",")
 	}
 
+	if pids, _ := procfs.PidOf("systemd-resolved"); len(pids) > 0 {
+		// procfs.PidOf only returns an error if the regex is empty or doesn't compile, so we can ignore it
```

Comment:
IMO - we should still emit a message as a breadcrumb to the user in case anything happens.